front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -60,9 +69,32 @@ public StorageServiceSupport(ObjectType objectType,
     if (refreshIntervalMs >= HEALTH_MILLIS) {
       throw new IllegalArgumentException("Cache refresh time must be more frequent than cache health timeout");
     }
+    String typeName = objectType.name();
     this.registry = registry;
+
+    this.cacheRefreshTimer = registry.timer(
+      registry.createId("storageServiceSupport.cacheRefreshTime", "objectType", typeName));
```

Comment:
should these metrics be prefixed with the component, i.e. "front50.storageServiceSupport.cacheRefreshTime" ? Is that a pattern followed by other components?